redaxo/src/core/lib/setup/setup.php

```diff
@@ -237,4 +237,37 @@ public static function checkDbSecurity()
 
         return $security;
     }
+
+    /**
+     * Returns true when we are running the very first setup for this instance.
+     * Otherwise false is returned, e.g. when setup was re-started from the core/systems page.
+     *
+     * @return bool
+     */
+    public static function isInitialSetup():bool {
+        $user_sql = rex_sql::factory();
+        $user_sql->setQuery('select * from ' . rex::getTable('user') . ' LIMIT 1');
```

Comment:
hier verwenden wir den gleichen indikator wie in step6, wo wir - wenn schon user im system bekannt - nicht erneut erwzingen dass ein user angelegt werden muss

Reply:
Ich vermute, dass es hier einen Fehler geben wird beim initialen Setup, wenn noch keine gültige DB-Verbindung hinterlegt ist.
Habe es aber nicht getestet.

Reply:
sollte jetzt behandelt sein